hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/event/remote/MultiActionCancelEvent.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2020 Bosch.IO GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository.event.remote;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.eclipse.hawkbit.repository.model.Action;
+import org.eclipse.hawkbit.repository.model.ActionProperties;
+import org.eclipse.hawkbit.repository.model.Target;
+
+/**
+ * Generic deployment event for the Multi-Assignments feature. The event extends
+ * the {@link MultiActionEvent} and holds a list of controller IDs to identify
+ * the targets which are affected by a deployment action and a list of
+ * {@link ActionProperties} containing information's about the affected actions
+ * as payload. This event is only published in case of an cancellation.
+ */
+public class MultiActionCancelEvent extends MultiActionEvent {
+
+    private static final long serialVersionUID = 1L;
+
+    private List<ActionProperties> actionProperties;
```

Comment:
`MultiActionCancelEvent` and `MultiActionAssignEvent` are both having `actionProperties`, does it make sense to move this to `MultiActionEvent` and make `MultiActionEvent` abstract? I don't see `actionProperties` be in use somewhere. Maybe the action status is the only thing which you require? If so I'd prefer to keep the event as small as possible since will be passed around a lot